pages/index.js

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import styled from 'styled-components';
 import TodoList from '../components/TodoList';
-import { CenteredContainer, AppTitle, ClearAllButton } from '../components/styled-components';
+import {
+  CenteredContainer,
+  AppTitle,
+  ClearAllButton
+} from '../components/styled-components';
 
 const HomePage = () => {
   return (
     <CenteredContainer>
-      <AppTitle>My To-Do App</AppTitle>
+      <AppTitle style={{ margin: '0' }}>My To-Do App</AppTitle>
```

Comment:
Try to avoid inline styles. They're like dynamite, bombing every style that might have been applied somewhere. Use props if you need to adjust individual attributes.